games-arcade/blobbyvolley2/blobbyvolley2-1.1~git.recipe

```diff
@@ -0,0 +1,77 @@
+SUMMARY="Official continuation of the famous Blobby Volley 1.x arcade game"
+DESCRIPTION="The head-to-head multiplayer ball game
+
+The \"Dedicated Server\" runs with a Gamespeed of 100%, which means 75 FPS
+The Port for the Server is 1234."
+HOMEPAGE="https://github.com/danielknobe/blobbyvolley2"
+COPYRIGHT="2006-2020 Daniel Knobe
+	2010-2015 Erik Schultheis
+	2006-2009 Jonathan Sieber
+	2006 Sven Rech"
+LICENSE="GNU GPL v2"
+REVISION="1"
+srcGitRev="70ec84da660bf59fe36e3d67508d009afa0ae9c8"
+SOURCE_URI="$HOMEPAGE/archive/$srcGitRev.tar.gz"
+CHECKSUM_SHA256="917291b4140f573030d63132860cde7cdacae4642efb6adf2abb754cb2c84afe"
+SOURCE_FILENAME="blobbyvolley2-$srcGitRev.tar.gz"
+SOURCE_DIR="blobbyvolley2-$srcGitRev"
+PATCHES="blobbyvolley2-$portVersion.patchset"
+
+ARCHITECTURES="!x86_gcc2 x86 x86_64"
+commandBinDir=$binDir
+if [ "$targetArchitecture" = x86_gcc2 ]; then
+SECONDARY_ARCHITECTURES="x86"
+commandBinDir=$prefix/bin
+fi
+
+PROVIDES="
+	blobbyvolley2$secondaryArchSuffix = $portVersion
+	app:Blobby = $portVersion
+	cmd:blobby_server
+	"
+REQUIRES="
+	haiku$secondaryArchSuffix
+	lib:libGL$secondaryArchSuffix
+	lib:libphysfs$secondaryArchSuffix
+	lib:libSDL2_2.0$secondaryArchSuffix
+	"
+
+BUILD_REQUIRES="
+	haiku${secondaryArchSuffix}_devel
+	devel:libboost_system$secondaryArchSuffix
+	devel:libGL$secondaryArchSuffix
+	devel:libphysfs$secondaryArchSuffix
+	devel:libSDL2$secondaryArchSuffix
```

Comment:
Should be libSDL2_2.0 (same as in lib:*)

Reply:
So that should be for all of these? If so, it seems like the ones in my previous PRs slipped by.

Reply:
Well, the devel package for libSDL2 provides "libSDL2.so", but we like to have them listed the same as for the base package: issue was created for these at: https://github.com/haikuports/haikuports/issues/3878

Reply:
I was confused by:

> These recipes should be changed to use lib:libSDL_1.2 and devel:libSDL instead

I guess it's just a general stylistic choice to match them and not specific to the issue?

Reply:
Maybe I should change that as we do for libpng16 :)

Reply:
Thoughts on this @korli @pulkomandy ?
@pulkomandy replied that having devel:libSDL2 could be OK if the devel package provides it?
```
If there is a libsdl_image.so in the develop/lib dir, then we need devel:libsdl_image
```

Reply:
Updated the description for the issue